mods/battle-royale/modules/utils/PseudoRandom.lua

```diff
@@ -21,8 +21,14 @@ function PseudoRandom:GetValue()
     -- if not, we shuffle!
     local n = table.getn(self.collection);
     if self.index == n then
+        local lastSide = self.collection[self.index];
```

Comment:
After shrinking 8 times then every side should've shrinken twice, no more and no less. I'm trying to think if that still holds with this change - my initial feeling says it does.